client/src/App.js

```diff
@@ -22,7 +22,10 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <div>
+          <Route exact path="/" component={MovieList} />
+          <Route path="/movies/:id" render={ (props) => <Movie {...props} addToSavedList={this.addToSavedList} />} />
```

Comment:
Excellent! Glad we were able to get this figured out :+1: Let me know if you have questions about it, but don't worry if it's still not 100% making sense right now. This is more advanced stuff, and once we learn Redux it will be a little less relevant :)